tests/OBO.NET.Tests/OboOntology.Tests.fs

```diff
@@ -171,4 +178,15 @@ module OboOntologyTests =
                     let expected = seq {Exact, testTerm5, Some testTerm1; Broad, testTerm5, Some testTerm2; Narrow, testTerm5, None}
                     Expect.sequenceEqual actual expected "is not equal"
             ]
+
+            testList "AreTermsEquivalent" [
+                testCase "checks equivalence correctly" <| fun _ ->
+                    Expect.isTrue (testOntology.AreTermsEquivalent(testTerm4, testTerm6)) "is not equal"
+            ]
+
+            testList "ReturnAllEquivalentTerms" [
+                testCase "returns correct terms" <| fun _ ->
+                    let actual = testOntology.ReturnAllEquivalentTerms(testOntology2)
+                    Expect.equal (Seq.head actual).Id testTerm4.Id "is not equal"

```

Comment:
Since you are testing a sequence here, i would test the whole sequence instead of just the head

Reply:
Then it would compare objects and they are not equal (due to reference). Or does Expecto circumvent this by checking by fields via reflection? 🤔 
(I'm not sure but) I think not.
Gonna test it.

Reply:
There is `Expect.SequenceEqual`.

Reply:
That's what I'm talkin' 'bout. 😄